test/swim_test.js

```diff
@@ -155,24 +156,25 @@ testRingpop('suspicion subprotocol cannot be reenabled without all timers first
     assert.equals(suspicion.isStoppedAll, 'fakestopall', 'suspicion not reenabled');
 });
 
-testRingpop({
-    async: true
-}, 'marks member faulty after suspect period', function t(deps, assert, done) {
+testRingpop('marks member faulty after suspect period', function t(deps, assert) {
     assert.plan(1);
 
+    // Add member, make it alive
     var membership = deps.membership;
-    var suspicion = deps.suspicion;
-
     var address = '127.0.0.1:3001';
     membership.makeAlive(address, Date.now());
 
-    var member = membership.findMemberByAddress(address);
-
+    var suspicion = deps.suspicion;
     suspicion.period = 1;
+
```

Comment:
Apologies for the code-shuffling. But I think this order of statements makes it more readable. This has nothing to do with the flappiness though.